### PR TITLE
Use GeoIP legacy when building GoAccess

### PR DESCRIPTION
**Pull request by allinurl**
Ensure GoAccess builds with GeoIP legacy support, removed unnecessary dependency and updated license.

### CODE REVIEW

File: main/goaccess/APKBUILD [hunk 1]
```diff
@@ -28,7 +28,7 @@ build() {
 		--mandir=/usr/share/man \
 		--infodir=/usr/share/info \
 		--localstatedir=/var \
-		--enable-geoip \
+		--enable-geoip=legacy \
```
Comment (allinurl) [inline main/goaccess/APKBUILD]: @jirutka Right now the options are `legacy` which will utilize the original GeoIP databases and `mmdb` which will utilize the enhanced GeoIP2 databases.

Comment (kaniini) [inline main/goaccess/APKBUILD]: Would we not want to support the enhanced databases then?

Comment (allinurl) [inline main/goaccess/APKBUILD]: @kaniini Certainly, but I'm not entirely sure if Alpine has a GeoIP2 package?

Comment (allinurl) [inline main/goaccess/APKBUILD]: Any updates on this? Thanks.

Comment (jirutka) [inline main/goaccess/APKBUILD]: It seems that we don’t have geoip 2.x yet.

**Comment by jirutka** [comment]
**Note for myself:** The source repository is gone, thus https://github.com/alpinelinux/aports/pull/1257.patch does not work. We must figure out how to get these patches or recreate them.

**Pull request closed without merging**